frontend/src/pages/account/Create.vue

```diff
@@ -143,6 +146,14 @@ export default {
         'input.password': function (v) {
             if (this.errors.password && v.length >= 8) {
                 this.errors.password = ''
+            } else {
+                this.errors.password = 'Password needs to be longer than 8 chars'
+                return
+            }
+            if (zxcvbn(v).score >= 2) {
+                this.errors.password = ''
+            } else {
+                this.errors.password = 'Password needs to be more complex'
```

Comment:
Don't think the logic is right here.

As I type I'm seeing the 'longer than 8 chars' message toggle on and off with each character.

Reply:
Should be fixed now